SignUpValidation/sample.dat

```diff
@@ -0,0 +1,14 @@
+{
+  "ui_locales": "en-US",
+  "email": "example@gmail.ec",
+  "surname": "Test User",
+  "displayName": "Test User",
+  "givenName": "Test User",
+  "identities": [
+    {
+      "signInType": "federated",
+      "issuer": "google.com",
+      "issuerAssignedId": "102373679322388401018"
+    }
+  ]
+}
```

Comment:
What is this file intended for?

Reply:
[`sample.dat` is primarily for Azure Portal web interface usage](https://github.com/Azure/Azure-Functions/issues/1241)
It's used when we want to try the function on Azure
![image](https://user-images.githubusercontent.com/20668624/109028639-010e1380-7690-11eb-803f-0148241457c8.png)